### DISCUSSION TITLE
fail to aquire ssl on second request

### DISCUSSION THREAD

**Discussion by Salpertio** [ZoeyVid/NPMplus#605] [Q&A] [answered]
raspberry pi - docker stack launched with the default compose.yaml

this seems to also happen while on unraid. wanted to scratch the itch and check before i drop a ticket. 

-------------------------------------
nginx: [warn] "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
nginx: [warn] "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
nginx: [warn] "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
nginx: [warn] "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
[Global   ] › ℹ  info      Using Sqlite: /data/etc/npm/database.sqlite
[Migrate  ] › ℹ  info      Current database version: none
[IP Ranges] › ℹ  info      Fetching IP Ranges from online services...
[IP Ranges] › ℹ  info      Fetching https://ip-ranges.amazonaws.com/ip-ranges.json
[IP Ranges] › ℹ  info      Fetching https://www.cloudflare.com/ips-v4
[IP Ranges] › ℹ  info      Fetching https://www.cloudflare.com/ips-v6
[SSL      ] › ℹ  info      Certbot Renewal Timer initialized
[IP Ranges] › ℹ  info      IP Ranges Renewal Timer initialized
[Global   ] › ℹ  info      Backend PID 480 listening on port 48693 ...
[Nginx    ] › ℹ  info      Starting Nginx
2024/01/23 02:40:39 [warn] 502#502: "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
2024/01/23 02:40:39 [warn] 502#502: "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
2024/01/23 02:40:39 [warn] 502#502: "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
2024/01/23 02:40:39 [warn] 502#502: "ssl_stapling" ignored, issuer certificate not found for certificate "/data/tls/dummycert.pem"
2024/01/23 02:40:39 [notice] 502#502: ModSecurity-nginx v1.0.3 (rules loaded inline/local/remote: 0/14/0)
2024/01/23 02:40:39 [alert] 502#502: worker process 507 exited on signal 11 (core dumped)
[Nginx    ] › ⬤  debug     Deleting file: /data/nginx/proxy_host/2.conf
[Nginx    ] › ⬤  debug     Could not delete file: {
  "errno": -2,
  "code": "ENOENT",
  "syscall": "unlink",
  "path": "/data/nginx/proxy_host/2.conf"
}
[Nginx    ] › ⬤  debug     Deleting file: /data/nginx/proxy_host/2.conf
[Nginx    ] › ⬤  debug     Could not delete file: {
  "errno": -2,
  "code": "ENOENT",
  "syscall": "unlink",
  "path": "/data/nginx/proxy_host/2.conf"
}
[Nginx    ] › ⬤  debug     Deleting file: /data/nginx/proxy_host/2.conf.err
[Nginx    ] › ⬤  debug     Could not delete file: {
  "errno": -2,
  "code": "ENOENT",
  "syscall": "unlink",
  "path": "/data/nginx/proxy_host/2.conf.err"
}
[Nginx    ] › ℹ  info      Reloading Nginx
2024/01/23 02:40:59 [alert] 502#502: worker process 699 exited on signal 11 (core dumped)
[Nginx    ] › ⬤  debug     Deleting file: /data/nginx/proxy_host/2.conf
[Nginx    ] › ⬤  debug     Could not delete file: {
  "errno": -2,
  "code": "ENOENT",
  "syscall": "unlink",
  "path": "/data/nginx/proxy_host/2.conf"
}
[Nginx    ] › ℹ  info      Reloading Nginx
[Nginx    ] › ⬤  debug     Deleting file: /data/nginx/proxy_host/2.conf
[Nginx    ] › ⬤  debug     Could not delete file: {
  "errno": -2,
  "code": "ENOENT",
  "syscall": "unlink",
  "path": "/data/nginx/proxy_host/2.conf"
}
[Nginx    ] › ⬤  debug     Deleting file: /data/nginx/proxy_host/2.conf
[Nginx    ] › ⬤  debug     Could not delete file: {
  "errno": -2,
  "code": "ENOENT",
  "syscall": "unlink",
  "path": "/data/nginx/proxy_host/2.conf"
}
[Nginx    ] › ⬤  debug     Deleting file: /data/nginx/proxy_host/2.conf.err
[Nginx    ] › ⬤  debug     Could not delete file: {
  "errno": -2,
  "code": "ENOENT",
  "syscall": "unlink",
  "path": "/data/nginx/proxy_host/2.conf.err"
}
[Nginx    ] › ℹ  info      Reloading Nginx
[SSL      ] › ℹ  info      Testing http challenge for xxxx
[SSL      ] › ℹ  info      Renewing Certbot certificates for Cert #2: xxxx
[SSL      ] › ℹ  info      Command: certbot --logs-dir /tmp/certbot-log --work-dir /tmp/certbot-work --config-dir /data/tls/certbot renew --force-renewal --config "/data/tls/certbot/config.ini" --cert-name "npm-2" --preferred-challenges "dns,http" --no-random-sleep-on-renew
[Express  ] › ⚠  warning   Saving debug log to /tmp/certbot-log/letsencrypt.log
No certificate found with name npm-2 (expected /data/tls/certbot/renewal/npm-2.conf).
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /tmp/certbot-log/letsencrypt.log or re-run Certbot with -v for more details.
2024/01/23 02:41:48 [alert] 502#502: worker process 1045 exited on signal 11 (core dumped)

**Comment by Zoey2936** [comment]
It seems like this cert does not exist, can you try to recrrate it?

**Comment by Zoey2936** [comment, marked as answer]
And please run `ls /opt/npm/tls/certbot/* -lhA` and send me the output - also please send me your compose.yaml
